### PR TITLE
[doc]Describe what to do when pre-adoption-validation fails

### PR DESCRIPTION
**Pull request by gibizer**
Implements: OSPRH-6875

### CODE REVIEW

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc [hunk 1]
```diff
@@ -613,6 +613,21 @@ oc logs -l app=openstackansibleee -f --max-log-requests 20
 oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
 
+. If the validation deployment failed then check the ansible execution logs to
```
Comment (marios) [inline docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:616]: if any openstack-pre-adoption validations fail, you must first determine which ones were unsuccessful and then follow the instructions below for each case:

(just a suggestion feel free to adopt part/all/none)

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc [hunk 2]
```diff
@@ -613,6 +613,21 @@ oc logs -l app=openstackansibleee -f --max-log-requests 20
...
+* if the hostname validation failed then check that the hostname of the EDPM
+node is correctly listed in the `OpenStackDataPlaneNodeSet`
+
+* if the kernel argument check failed then make sure that the
+`OpenStackDataPlaneNodeSet` has the same kernel argument configuration in
+`edpm_kernel_args` and `edpm_kernel_hugepages` variables than what is used in
+the 17 node.
+
+* if the tuned profile check failed then make sure that the
+`edpm_tuned_profile` variable in the `OpenStackDataPlaneNodeSet` configured to
```
Comment (marios) [inline docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:628]: is configured to

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc [hunk 3]
```diff
@@ -613,6 +613,21 @@ oc logs -l app=openstackansibleee -f --max-log-requests 20
...
+use the same profile as it used on the 17 node.
```
Comment (marios) [inline docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:629]: same profile as set on the (source) OSP 17 node.

**Review by marios — COMMENTED** [review]
thanks for posting it - just some inline suggestions but overall lgtm

wondering if we need to tell them more about 'how to determine which validation failed'

**Comment by gibizer** [comment]
> thanks for posting it - just some inline suggestions but overall lgtm
> 
> wondering if we need to tell them more about 'how to determine which validation failed'

How to get the ansible logs is describe just above this. How to understand ansible logs can be an interesting doc topic, but I think it is out of scope of this patch as I think it requires a more general approach.

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc (hunk 1 above)
Comment (gibizer) [reply docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:616]: applied your suggestion with a small tweak

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc (hunk 2 above)
Comment (gibizer) [reply docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:628]: Done

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc (hunk 3 above)
Comment (gibizer) [reply docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:629]: Applied

**Review by jistr — APPROVED** [review]

**Pull request merged**